libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml

```diff
@@ -8,13 +8,13 @@
   
   <table><thead><tr>
   
-    <td ><b>Isin</b></td>
+    <td style="text-align:center; width:10%" ><b>Isin</b></td>
```

Comment:
ISIN is a string, it shouldn't be centered.

Reply:
Done. Kept the String representation as like older one